src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/BespokeCondition.kt

```diff
@@ -15,8 +17,9 @@ data class BespokeCondition(
   @NotNull
   val id: Long = -1,
 
-  @NotNull
-  val licenceId: Long = -1,
+  @ManyToOne
+  @JoinColumn(name = "licence_id", nullable = false)
+  var licence: Licence,
```

Comment:
Set up proper hibernate relationships between entities for much easier orphan removal and cacade updates